.github/actions/scenarios/dubbo/lane-dubbo3/action.yml

```diff
@@ -0,0 +1,285 @@
+        DUBBO_PROTOCOL_PORT: 18821
+        DUBBO_REGISTRY_MODE: interface
+      run: |
+        nohup java -javaagent:sermant-agent-${{ env.sermantVersion }}/agent/sermant-agent.jar=appName=dubbo-integration-provider -Dsermant_log_dir=${{ env.logDir }}/provider-${{ matrix.dubbo-version }}-${{ matrix.dubbo-versions }} -jar \
+        sermant-integration-tests/dubbo-test/dubbo-${{ env.projectPath }}-integration-provider/target/dubbo-integration-provider.jar > ${{ env.logDir }}/dubbo-provider.log 2>&1 &
+    - name: start provider service instance
+      shell: bash
+      if: matrix.registry-model == 'instance'
+      env:
+        dynamic.config.dynamicConfigType: KIE
+        dynamic.config.serverAddress: 127.0.0.1:30110
+        SERVICE_META_ENVIRONMENT: development
+        SERVICE_META_APPLICATION: lane
+        SERVICE_META_VERSION: 1.0.0
+        SERVER_PORT: 18021
+        DUBBO_PROTOCOL_PORT: 18821
+        DUBBO_REGISTRY_MODE: instance
+      run: |
+        nohup java -javaagent:sermant-agent-${{ env.sermantVersion }}/agent/sermant-agent.jar=appName=dubbo-integration-provider -Dsermant_log_dir=${{ env.logDir }}/provider-${{ matrix.dubbo-version }}-${{ matrix.dubbo-versions }} -jar \
+        sermant-integration-tests/dubbo-test/dubbo-${{ env.projectPath }}-integration-provider/target/dubbo-integration-provider.jar > ${{ env.logDir }}/dubbo-provider.log 2>&1 &
+    - name: start second provider service all
+      shell: bash
+      if: matrix.registry-model == 'all'
+      env:
+        dynamic.config.dynamicConfigType: KIE
+        dynamic.config.serverAddress: 127.0.0.1:30110
+        SERVICE_META_ENVIRONMENT: development
+        SERVICE_META_APPLICATION: lane
+        SERVICE_META_VERSION: 1.0.1
+        SERVER_PORT: 18022
+        DUBBO_PROTOCOL_PORT: 18822
+        DUBBO_REGISTRY_MODE: all
+      run: |
+        nohup java -javaagent:sermant-agent-${{ env.sermantVersion }}/agent/sermant-agent.jar=appName=dubbo-integration-provider -Dsermant_log_dir=${{ env.logDir }}/provider2-${{ matrix.dubbo-version }}-${{ matrix.dubbo-versions }} -jar \
+        sermant-integration-tests/dubbo-test/dubbo-${{ env.projectPath }}-integration-provider/target/dubbo-integration-provider.jar > ${{ env.logDir }}/dubbo-provider2.log 2>&1 &
+    - name: start second provider service interface
+      shell: bash
+      if: matrix.registry-model == 'interface'
+      env:
+        dynamic.config.dynamicConfigType: KIE
+        dynamic.config.serverAddress: 127.0.0.1:30110
+        SERVICE_META_ENVIRONMENT: development
+        SERVICE_META_APPLICATION: lane
+        SERVICE_META_VERSION: 1.0.1
+        SERVER_PORT: 18022
+        DUBBO_PROTOCOL_PORT: 18822
+        DUBBO_REGISTRY_MODE: interface
+      run: |
+        nohup java -javaagent:sermant-agent-${{ env.sermantVersion }}/agent/sermant-agent.jar=appName=dubbo-integration-provider -Dsermant_log_dir=${{ env.logDir }}/provider2-${{ matrix.dubbo-version }}-${{ matrix.dubbo-versions }} -jar \
```

Comment:
check all the server_port. It should not be duplicate

Reply:
fixed